test/fail_compilation/fail262.d

```diff
@@ -1,7 +1,7 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/fail262.d(23): Error: function `fail262.B.f` does not override any function
+fail_compilation/fail262.d(23): Error: function `const void fail262.B.f()` does not override any function, did you mean to override `shared const void fail262.A.f()`?
```

Comment:
Awesome improvement, but looks unrelated to this ? Or does the error improve as a result of changing a code path?

Reply:
Yes, the error message changes due to a change in the code path.